components/molecules/__snapshots__/PendingBenefitDetails.test.js.snap

```diff
@@ -0,0 +1,398 @@
+                >
+                  <div
+                    class="w-3 h-3 mt-2 rounded-full ml-1/5 bg-green-active"
+                  />
+                  <div
+                    class="w-0.5 mt-1.5 ml-[5px] h-[92%] bg-green-active"
+                  />
+                </div>
+                <div
+                  class="ml-3 "
+                >
+                  <p
+                    class="text-md font-bold"
+                  >
+                    Documents reviewed
+                  </p>
+                  <p
+                    class="text-sm"
+                  >
+                    September 8, 2021
+                  </p>
+                </div>
+              </div>
+              <div
+                class="py-2.5 flex w-60"
+              >
+                <div
+                  class="v-stepper w-5"
+                >
+                  <div
+                    class="w-3 h-3 mt-2 rounded-full ml-1/5 bg-green-active"
+                  />
+                  <div
+                    class="w-0.5 mt-1.5 ml-[5px] h-[92%] bg-green-active"
+                  />
+                </div>
+                <div
+                  class="ml-3 "
+                >
+                  <p
+                    class="text-md font-bold"
+                  >
+                    Documents uploaded
+                  </p>
+                  <p
+                    class="text-sm"
+                  >
+                    September 6, 2021
+                  </p>
+                </div>
+              </div>
+              <div
+                class="py-2.5 flex w-60"
+              >
+                <div
+                  class="v-stepper w-5"
+                >
+                  <div
+                    class="w-3 h-3 mt-2 rounded-full ml-1/5 bg-green-active"
+                  />
+                  <div
+                    class="w-0.5 mt-1.5 ml-[5px] h-[92%] bg-green-active"
+                  />
+                </div>
+                <div
+                  class="ml-3 "
+                >
+                  <p
+                    class="text-md font-bold"
+                  >
+                    Documents requested
+                  </p>
+                  <p
+                    class="text-sm"
+                  >
+                    September 5, 2021
+                  </p>
+                </div>
+              </div>
+              <div
+                class="pb-2.5 flex mt-0 h-max"
+              >
+                <div
+                  id="v-stepper"
+                >
+                  <div
+                    class="w-0.5 mt-1.5 ml-[5px] h-[92%] bg-green-active"
+                  />
+                </div>
+                <div
+                  class="mt-1 ml-6"
+                  id="stepper-content"
+                >
+                  <a
+                    class="text-xs text-left text-bright-blue-solid underline"
+                    href="./dashboard"
+                  />
```

Comment:
I'll approve for now but instead of anchor tags it is better to use next's provided Link (there are a few of theses)
https://nextjs.org/docs/api-reference/next/link

It stops hard refreshes and lets us easily use the built in routing